packages/backend-app-api/src/wiring/BackendInitializer.ts

```diff
@@ -336,6 +336,21 @@ export class BackendInitializer {
       await this.#serviceRegistry.get(coreServices.rootLogger, 'root'),
     );
 
+    const rootConfig = await this.#serviceRegistry.get(
+      coreServices.rootConfig,
+      'root',
+    );
+
+    const startupOptions =
+      rootConfig
+        ?.getOptionalConfig('backend.startup')
+        ?.get<BackendStartupOptions>() ?? {};
```

Comment:
Avoid using `get()` since it doesn't give as good feedback for invalid config or will allow invalid config. Switching this to `.getOptionalBoolean(...)` would give a clean error message if there's an unexpected value in config.